File: travelAgency2/travelAgency2/Domain/Purchase.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace travelAgency2.src.Domain
{
    internal class Purchase : Entity<long>
    {

        public Flight flight { get; set; }
        public string clientName { get; set; }
        public string clientAddress { get; set; }
        public List<string> tourists { get; set; }
        public int nrOfSeats { get; set; }

        public Purchase(long id, Flight flight, string clientName, string clientAddress, List<string> tourists, int nrOfSeats) : base(id)
        {
            this.flight = flight;
            this.clientName = clientName;
            this.clientAddress = clientAddress;
            this.tourists = tourists;
            this.nrOfSeats = nrOfSeats; 
        }
    }
}

File: travelAgency/src/main/java/group/travelagency/domain/Purchase.java
package group.travelagency.domain;

import java.util.List;

public class Purchase extends Entity<Long> {
    private Flight flight;
    private String clientName;
    private String clientAddress;
    private List<String> tourists;
    private int nrOfSeats;

    public Purchase(Flight flight, String clientName, String clientAddress,
                    List<String> tourists, int nrOfSeats) {
        this.flight = flight;
        this.clientName = clientName;
        this.clientAddress = clientAddress;
        this.tourists = tourists;
        this.nrOfSeats = nrOfSeats;
    }

    public Flight getFlight() {
        return flight;
    }

    public void setFlight(Flight flight) {
        this.flight = flight;
    }

    public String getClientName() {
        return clientName;
    }

    public void setClientName(String clientName) {
        this.clientName = clientName;
    }

    public String getClientAddress() {
        return clientAddress;
    }

    public void setClientAddress(String clientAddress) {
        this.clientAddress = clientAddress;
    }

    public List<String> getTourists() {
        return tourists;
    }

    public void setTourists(List<String> tourists) {
        this.tourists = tourists;
    }

    public int getNrOfSeats() {
        return nrOfSeats;
    }

    public void setNrOfSeats(int nrOfSeats) {
        this.nrOfSeats = nrOfSeats;
    }

    @Override
    public String toString() {
        return "Purchase{" +
                "flight=" + flight +
                ", clientName='" + clientName + '\'' +
                ", clientAddress='" + clientAddress + '\'' +
                ", tourists=" + tourists +
                ", nrOfSeats=" + nrOfSeats +
                '}';
    }
}
